modyn/config/schema/pipeline/evaluation/trigger/periodic.py

```diff
@@ -0,0 +1,31 @@
+from functools import cached_property
+from typing import Literal
+
+from modyn.const.regex import REGEX_TIME_UNIT
+from modyn.utils.utils import SECONDS_PER_UNIT
+from pydantic import Field
+
+from .base import MatrixEvalTriggerConfig
+
+
+class PeriodicEvalTriggerConfig(MatrixEvalTriggerConfig):
+    """Comparable to TimeTriggerConfig and DataAmountTriggerConfig, but for scheduling evaluation runs.
+
+    Note: This strategy will run evaluations after the core pipeline.
```

Comment:
Here for example. This is a trigger. But is it also a strategy? I dont understand the difference